lib/src/git.rs

```diff
@@ -325,6 +325,8 @@ pub fn export_refs(
 pub enum GitFetchError {
     #[error("No git remote named '{0}'")]
     NoSuchRemote(String),
+    #[error("Invalid glob provided. Globs may not contain the `:` character")]
+    InvalidGlob,
```

Comment:
Since several globs can be given, maybe embed the problematic one in the error.

Reply:
I think it should be obvious enough which of the globs contains these characters.